static/js/biomedical/disease/chart.ts

```diff
@@ -247,3 +256,78 @@ export function drawDiseaseSymptomAssociationChart(
       (d) => `Symptom: ${d.name}<br>Odds Ratio Association: ${d.oddsRatio}`
     );
 }
+/**
+ * Draws the disease ontology hierarchy chart for the disease of interest
+ * @param id the div id where the chart is rendered on the page
+ * @param data the disease data passed into the function
```

Comment:
nit: since we've specified that this is a param, we already know that this is data passed into the function. Instead, we should describe what the data is/how it is used. Something like "data to show in the chart for" Maybe something about the fact that this is just the parent node which contains information about all children nodes

Reply:
Good point! Done for the disease ontology hierarchy chart. Would you want me to do the same for the charts we made in the past?